_posts/2017-09-28-our-engineering-culture-methodologies-and-philosophies.md

```diff
@@ -0,0 +1,179 @@
+into the system.
+- https://www.atlassian.com/agile/code-reviews
+- https://hackernoon.com/the-art-of-pull-requests-6f0f099850f9
+- https://medium.com/@vaidehijoshi/crafting-better-code-reviews-1a5fc00a9312
+  
+## Continuous Delivery  
+There are a lot of pros and cons around having a Continuous Delivery setup. We
+have most of our applications (all but our application that handles money) on
+continuous delivery. This methodology removes the ceremony around deployment.
+Removing the deployment ceremony reduces engineer frustration, increases
+engineer productivity, and so much more. From a business standpoint, features
+get released faster and quicker.
+
+Here are some articles about what continuous delivery is and it's benefits to an
+organization.
+- https://martinfowler.com/bliki/ContinuousDelivery.html
+
+## Pair Programming  
+It's difficult to concisely speak about the importance of pair programming.
+Beyond the immediate impact it provides engineers, it also has long lasting
+effects that can't be replaced with other frameworks. It helps form bonds of
+mentorship and coaching as to improving the engineer that requested the pair. It
+further strengthens trust and bonds between individuals as they build a rapport
+and help each other through challenges.
+
+While there are plenty of pro and con articles around the internet about
+pairing, here are some about the positive benefits of it and why we use it
+at Forward Financing:
+- https://raygun.com/blog/how-good-is-pair-programming-really/
+  
+## Testing  
+Testing is another important vehicle that helps reduce technical debt from
+being introduced into new systems and helps vet that requirements are met. Tests
+are a way an engineer can programatically understand if it's functionality is
+in fact doing what is intended.
+
+Here are some articles about why testing is important and why companies decide
+to invest in testing.
+- https://martinfowler.com/bliki/UnitTest.html
+  
+## Continuous Learning  
+Continuous learning is nothing new to businesses, but has made a recent
+resurgence in business culture. This framework allows companies to invest in
+employees by giving them new skills, reinforcing old skills with new twists, and
+more. This has so many positive effects on an organization that it is difficult
+to enumerate them all. Two that I tend to focus on are:
+1) Employee retention
+2) Employee growth.
+
+Here are some links around the internet that describe what continuous learning
+is and why it is important.
+- https://www.td.org/Publications/Magazines/TD/TD-Archive/2017/02/Webex-10-Ways-to-Build-a-Culture-of-Continuous-Learning
+  
+## Management  
+Managing a team of engineers is often likened to herding cats. It is a tall task
+that most people fumble through. Being the liason between senior management and
+engineers also has it's own set of challenges. The two most important pieces
+that any manager can do for engineers are:
```

Comment:
engineers: ~are:~
the verb negates the need for the colon, but having a bullet point list makes it easier to read.